src/libs/actions/Report.js

```diff
@@ -904,21 +919,27 @@ function addAction(reportID, text, file) {
  * network layer handle the delayed write.
  *
  * @param {Number} reportID
- * @param {Number} sequenceNumber
+ * @param {Number} [sequenceNumber]
  */
 function updateLastReadActionID(reportID, sequenceNumber) {
-    const currentMaxSequenceNumber = reportMaxSequenceNumbers[reportID];
-    if (sequenceNumber < currentMaxSequenceNumber) {
-        return;
-    }
+    // If we are not specifying the sequence number, let's set it to the max one available
```

Comment:
This is a pretty good comment, but I think it can be taken a step further to help with the context. If you add a "so that..." at the end of it, the comment then fully explains what is trying to be accomplished with the logic.